src/stores/ShardDetail.ts

```diff
@@ -166,9 +358,13 @@ export const getInitialState = (
 export const shardDetailSelectors = {
     shards: (state: ShardDetailStore) => state.shards,
     setShards: (state: ShardDetailStore) => state.setShards,
-    getShardStatusColor: (state: ShardDetailStore) => state.getShardStatusColor,
-    getShardStatus: (state: ShardDetailStore) => state.getShardStatus,
+    getTaskShards: (state: ShardDetailStore) => state.getTaskShards,
+    getTaskShardDetails: (state: ShardDetailStore) => state.getTaskShardDetails,
+    getTaskStatusColor: (state: ShardDetailStore) => state.getTaskStatusColor,
     getShardDetails: (state: ShardDetailStore) => state.getShardDetails,
+    getShardStatusColor: (state: ShardDetailStore) => state.getShardStatusColor,
+    getShardStatusMessageId: (state: ShardDetailStore) =>
+        state.getShardStatusMessageId,
     evaluateShardProcessingState: (state: ShardDetailStore) =>
         state.evaluateShardProcessingState,
```

Comment:
Not for this PR... I am liking a different selector pattern... where we write selectors as "groups" on objects:

```
shards: {
  "get": () => ...,
  "set": () => ...,
  "reset": : () => ...,
  "foo": : () => ...,
},
"taskShards": {...},
"shardDetails": {...},
...
```

Granted  I have really like this pattern when fetching _JSON Forms_ data where it is inside of nested object `data`... so not sure how I'd like it with a more standard store like this one.